src/utils/components/KubevirtFilterToolbar/KubevirtFilterToolbar.tsx

```
@@ -24,8 +27,8 @@ type KubevirtFilterToolbarProps = {
   filters: KubevirtFilterState;
   hideColumnManagement?: boolean;
   loaded?: boolean;
-  onSetFilters: (newFilters: Partial<KubevirtFilterState>) => void;
   toolbarEndContent?: ReactNode;
+  onSetFilters: OnSetFilters;
```

ESLint

[error] Expected "onSetFilters" to come before "toolbarEndContent"

(perfectionist/sort-object-types)